src/components/AnimateBlock/__tests__/AnimateBlock.test.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+
+import {render, screen} from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+import AnimateBlock from '../AnimateBlock';
+
+const qaId = 'animate-block';
+
+describe('AnimateBlock', () => {
+    test('render AnimateBlock by default', async () => {
+        render(<AnimateBlock qa={qaId} />);
+
+        const component = screen.getByTestId(qaId);
+        expect(component).toBeInTheDocument();
+    });
+
+    test('add className', () => {
+        const className = 'my-class';
+
+        render(<AnimateBlock qa={qaId} className={className} />);
+        const component = screen.getByTestId(qaId);
+
+        expect(component).toHaveClass(className);
+    });
+
+    test('add className', () => {
+        const className = 'my-class';
+
+        render(<AnimateBlock qa={qaId} className={className} />);
+        const component = screen.getByTestId(qaId);
+
+        expect(component).toHaveClass(className);
+    });
+
+    test('use passed style', () => {
+        const style = {color: 'red'};
```

Comment:
Maybe (I'm mot sure) it is also can be a common variable

Reply:
Let us leave it as it is